content/applications/hr/payroll/configuration.rst

```diff
@@ -0,0 +1,315 @@
+
+Salary *rule parameters* affect the various rules for salary configuration. New rule parameters are
+not recommended to create unless specifically needed by the accounting department. To access the
+parameters, go to :menuselection:`Payroll --> Configuration --> Rule Parameters`.
+
+Other input types
+-----------------
+
+When creating payslips, it is sometimes necessary to add other entries for specific circumstances,
+like expenses, reimbursements, or deductions. These other inputs can be seen by going to
+:menuselection:`Payroll --> Configuration --> Other Input Types`.
+
+.. image:: configuration/other-input.png
+   :align: center
+   :alt: Other input types for payroll.
+
+To create a new *Input Type*, click the :guilabel:`Create` button. Enter the
+:guilabel:`description`, the :guilabel:`code`, and which structure type it applies to. Click the
+:guilabel:`Save` button to save the changes, or click :guilabel:`Discard` to delete the entry.
+
+.. image:: configuration/input-type-new.png
+   :align: center
+   :alt: Create a new Input Type.
+
+Salary package configurator
+===========================
+
+The various sections under the :guilabel:`Salary Package Configurator` section all affect an
+employee's potential salary. These sections specify what can be offered to an employee, and
+depending on what information an employee enters, their salary is adjusted accordingly (such as
+deductions, dependents, etc.). When an applicant applies for a job on the company website, the
+sections under Salary Package Configurator directly affect what the applicant sees, and what is
+populated as the applicant enters information.
+
+Advantages
+----------
+
+When offering potential employees a position, there can be certain advantages set in Odoo to make
+the offer more appealing other than the salary, such as extra time off, access to a company car,
+reimbursement for a phone or internet, etc. Any specific advantage is listed in the
+:guilabel:`Advantages` section of the configuration menu. To see the advantages, go to
+:menuselection:`Payroll --> Configuration --> Advantages`.
+
+.. image:: configuration/advantages.png
+   :align: center
+   :alt: Settings available for payroll.
+
+To make a new advantage, click the :guilabel:`Create` button. Enter the information in the fields,
+then click the :guilabel:`Save` button to save the changes, or click :guilabel:`Discard` to delete
+the entry.
+
+.. image:: configuration/new-advantage.png
+   :align: center
+   :alt: List of advantages employee's can have.
+
+Personal info
+-------------
+
+Every employee in Odoo has an `employee card` that includes all of their personal information,
+resume, work information, and documents.
+
+.. note::
+   An employee card can be thought of as an employee file.
+
+The *Personal Information* section lists all of the fields that are available to enter on the
+employee's card. To access this section, go to :menuselection:`Payroll --> Configuration -->
+Personal Information`.
```

Comment:
The UI actually reads "Personal Info" instead of "Personal Information", so I think we should go with the UI even though spelling out the word "information" is more proper.
```suggestion
Personal Info`.
```